### PR TITLE
zh-TW: Translation for src/android/aidl/changing.md

### PR DESCRIPTION
**Pull request by cafeholyc**

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -10265,14 +10265,14 @@ msgstr ""
...
-msgstr ""
+msgstr "# 改寫 API"
 
 #: src/android/aidl/changing.md:3
 msgid ""
 "Let us extend the API with more functionality: we want to let clients "
 "specify a\n"
 "list of lines for the birthday card:"
-msgstr ""
+msgstr "讓我們為這個API擴充更多功能: 我們想要讓用戶指定一系列的生日卡"
```
Comment (hueich) [inline po/zh-TW.po:10275]:
```suggestion
msgstr "讓我們為這個API擴充更多功能：我們想要讓用戶能在生日卡上指定幾行字："
```

**Comment by mgeisler** [comment]
Dear reviewers, please take a look at this PR so we can get it merged. Thanks!

File: po/zh-TW.po [hunk 2]
```diff
@@ -10265,14 +10265,14 @@ msgstr ""
 
 #: src/android/aidl/changing.md:1
 msgid "# Changing API"
-msgstr ""
+msgstr "# 改寫 API"
```
Comment (johnathan79717) [inline po/zh-TW.po:10268]: Could you also update  src/SUMMARY.md:196 please?

File: po/zh-TW.po [hunk 3]
```diff
@@ -10265,14 +10265,14 @@ msgstr ""
...
 
 #: src/android/aidl/changing.md:3
 msgid ""
 "Let us extend the API with more functionality: we want to let clients "
 "specify a\n"
 "list of lines for the birthday card:"
-msgstr ""
+msgstr "讓我們為這個API擴充更多功能：我們想要讓用戶能在生日卡上指定幾行字："
```
Comment (mingyc) [inline po/zh-TW.po:10275]: Could you please add spaces between english and Chinese characters?

**Review by mingyc — DISMISSED** [review]

File: po/zh-TW.po (hunk 2 above)
Comment (cafeholyc) [reply po/zh-TW.po:10268]: done

File: po/zh-TW.po (hunk 3 above)
Comment (cafeholyc) [reply po/zh-TW.po]: done

**Comment by mgeisler** [comment]
@mingyc and @hueich, this should be ready to go, I think the comments have been addressed.

**Comment by mgeisler** [comment]
@cafeholyc, thanks for the PR! Can you try rebasing it on top of the latest `main` and push the result here? That should trigger the tests again. Thanks!

**Comment by cafeholyc** [comment]
> @cafeholyc, thanks for the PR! Can you try rebasing it on top of the latest `main` and push the result here? That should trigger the tests again. Thanks!

@mgeisler, for some reason, the rebase button is disabled when I choose main branch. any idea?

Screenshot 2023-07-28 at 9 00 08 PM

**Comment by mgeisler** [comment]
Hi @cafeholyc, yeah, I think this is because you are comparing with the `main` branch in your fork (instead of the `main` branch in this repository).

When I go to my own fork in GitHub, I see that the `main` branch is behind:

image

The "Sync fork" button in the top-right can be used to update the `main` branch in the fork:

image

In your case, your `main` branch is both ahead and behind of the upstream `main` branch. This is likely because you've done the development on the `main` branch and then created the `aidl-zh-tw` later from the most recent commit on your `main`. Both branches point to the same commit ([dbd98e6](https://github.com/cafeholyc/comprehensive-rust/commit/dbd98e6741f30a9b3dc1f1de2172692e96aeb367)) right now.

You should still go and sync your fork and update your `main` branch to match the one in this repository. Your `aidl-zh-tw` branch will stay put at dbd98e6 and you will be able to merge `main` into it (or rebase it on top of `main`).

**Review by hueich — APPROVED** [review]

File: po/zh-TW.po [hunk 4]
```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
 "POT-Creation-Date: \n"
-"PO-Revision-Date: 2023-06-02 14:43+0800\n"
+"PO-Revision-Date: 2023-06-12 13:27+0800\n"
```
Comment (mgeisler) [inline po/zh-TW.po:5]: This change is just to kick off the tests again, it's not strictly necessary. However, updating the timestamp here is also not strictly necessary and in general, we get fewer merge conflicts if we don't update it:

```suggestion
"PO-Revision-Date: 2023-06-02 14:43+0800\n"
```

**Pull request merged**